packages/InputFeedback/InputFeedback.jsx

```diff
@@ -17,6 +17,7 @@ const InputFeedback = ({ feedback, children, ...rest }) => (
     {...safeRest(rest)}
     inset={3}
     dangerouslyAddClassName={feedback ? styles[feedback] : styles.default}
+    role={feedback === 'error' ? 'alert' : null}
```

Comment:
Considering this will override any consumer's custom usage of `role`, I don't consider this a breaking change since we originally intended to control the `role` attribute to match the component's state anyway.